src/main/java/dataloader/CsvDataLoader.java

```diff
@@ -0,0 +1,33 @@
+package dataloader;
+
+import org.apache.commons.csv.CSVFormat;
+import org.apache.commons.csv.CSVParser;
+import org.apache.commons.csv.CSVRecord;
+
+import java.io.IOException;
+import java.io.Reader;
+import java.io.StringReader;
+
+/**
+ * Simple class that parses data set from a csv file
+ * without knowing the headers of the file
+ */
+
+public class CsvDataLoader {
+
+    public void parseFile(String csvFile) throws IOException {
+
+        String location = "/path/to/file.csv";
```

Comment:
We don't need the ```location``` this should be specified in the ```csvFile``` input parameter.

Reply:
Think about how to account for the delimiter  :) :)

Reply:
provide an example here, please.
did you check the link I sent you on the apache csv library?

Reply:
just a glimpse of it  I would say :) :)

Reply:
delimiters are handled by the function itself.